tests/integration/test_interaction.py

```diff
@@ -562,6 +582,81 @@ def test_submit_button_shown(self):
 
         self.assertEqual(submit_button.get_attribute('disabled'), None)
 
+    def test_misplaced_items_returned_to_bank(self):
```

Comment:
@staubina Ok, but I would like to point out that most of the tests does not have docstrings and test name is intended to be self-explanatory.